libs/top_level_metrics.py

```diff
@@ -22,7 +22,8 @@
 RT_TRUNCATION_DAYS = 7
 
 
-MAX_METRIC_LOOKBACK_DAYS = 7
+# CMS and HHS testing data can both lag by more than 7 days. Let's use it unless it's 2 weeks old.
+MAX_METRIC_LOOKBACK_DAYS = 13
```

Comment:
cc/ @ghop02 FYI- I'm changing to allow for more lag, since HHS seems to lag by more than 7 days sometimes and we know CMS will lag by 7-10 days in many cases.